tests/test_models.py

```diff
@@ -29,3 +29,17 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+
+def test_daily_min_integers():
+    """Test that mean function works for an array of positive integers."""
+    from inflammation.models import daily_mean
+
+    test_input = np.array([[1, 2],
+                           [3, 4],
+                           [5, 6]])
+    test_result = np.array((1,2))
+    test_result[0] = 1
+    test_result[1] = 4
```

Comment:
```suggestion
    test_result = np.array([1,2])
```
Simpler assignment and correcting min value for column 2.

Reply:
Yes, should be 2 not 4. But clearly my method of working with numpy is right!

Reply:
OK, you know best!